File: Code Chef/JavaScript/400-499/410 Is it hot or cold HOTCOLD.js
process.stdin.setEncoding("utf-8");

const strToInt = (strArray) => {
    strArray.shift();
    const intArray = strArray.map(Number);
    
    return intArray;
};

const determineWeather = (temperature) => {
    (temperature > 20)
        ? console.log("HOT")
        : console.log("COLD");
};

process.stdin.on("data", (input) => {
    const testcases = strToInt(input.trim().split("\n"));
    
    testcases.forEach((temperature) => {
        determineWeather(temperature);
    });
});

File: Code Chef/C Programming/400-499/410 Is it hot or cold HOTCOLD.c
#include <stdio.h>
int main() 
{
    int t,i,c;
    scanf("%d",&t);
    for(i=0;i<t;i++)
    {
        scanf("%d",&c);
        if(c<=20)
        {
            printf("COLD\n");
        }
        else
        {
            printf("HOT\n");
        }
    }
	return 0;
}